pkg/deviceshifu/deviceshifuopcua/deviceshifuopcua.go

```diff
@@ -221,16 +221,16 @@ func (ds *DeviceShifu) requestOPCUANodeID(nodeID string) error {
 
 	resp, err := ds.opcuaClient.ReadWithContext(ctx, req)
 	if err != nil {
-		log.Printf("Failed to read message from Server, error: %v " + err.Error())
+		klog.Errorf("Failed to read message from Server, error: %v " + err.Error())
 		return err
 	}
 
 	if resp.Results[0].Status != ua.StatusOK {
-		log.Printf("OPC UA response status is not OK, status: %v", resp.Results[0].Status)
+		klog.Errorf("OPC UA response status is not OK, status: %v", resp.Results[0].Status)
 		return err
 	}
 
-	log.Println(resp.Results[0].Value.Value())
+	klog.Infoln(resp.Results[0].Value.Value())
```

Comment:
You used klog.Infoln here, and used klog.Infof at line 187 with exactly the same content. Please make the log consistent, don't use two different methods for same content. In fact, just use Infof, Errorf... etc for all logs. klog will and newline for you.

Reply:
Why is this resolved? @MrLeea-13155bc Please don't resolve the comment before addressing it.

Reply:
I still see a bunch of ```...ln(...)```

Reply:
forget to push  😢